lib/webauthn/authenticator_data/attested_credential_data.rb

```diff
@@ -54,14 +47,20 @@ def length
 
       attr_reader :data
 
+      def valid_credential_public_key?
+        cose_key = COSE::Key.deserialize(public_key)
+
+        cose_key.algorithm
```

Comment:
Almost cosmetic here, but given this method ends in a `?` I would expect to return a boolean.
What do you think about `!!cose_key.algorithm`?

Reply:
Right...

If we prefer that we should maybe have rubocop checking that..

Reply:
Never built a solid opinion on that particular pattern FWIW.

Reply:
Not a strong opinion neither. Specially not on how to implemented. I do like when something ends with a ? returns a boolean.

Your call 👍